lib/runtime/wazero/instance.go

```diff
@@ -108,7 +108,7 @@ func NewInstance(code []byte, cfg Config) (instance *Instance, err error) {
 
 	_, err = rt.NewHostModuleBuilder("env").
 		// values from newer kusama/polkadot runtimes
-		ExportMemory("memory", 23).
+		ExportMemory("memory", 24).
```

Comment:
Not sure about this value

Reply:
I used `2070` as the value of `memory pages` as I got the ans [here in Slack](https://chainsafeio.slack.com/archives/CEATMPULS/p1712414223681539)